From: Chad Dupuis <cdupuis@redhat.com>
Date: Fri, 20 Aug 2010 19:28:19 -0400
Subject: [scsi] qla4xxx: update version to 5.02.03.00.05.06-d1
Message-id: <20100820004100.6639.7275.sendpatchset@localhost.localdomain>
Patchwork-id: 27750
O-Subject: [RHEL 5.6 PATCH 2/2] qla4xxx: Update driver version to
	5.02.03.00.05.06-d1.
Bugzilla: 623675

Bugzilla
========

623675

Upstream Status
===============

This patch is RHEL 5 specific.

Description
===========

>From b5bb297c2514aee22d65b61017397705b88e9b7d Mon Sep 17 00:00:00 2001
From: root <root@labrat.lab.bos.redhat.com>
Date: Fri, 20 Aug 2010 09:25:18 -0400
Subject: [PATCH] qla4xxx: Update driver version to 5.02.03.00.05.06-d1.

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/scsi/qla4xxx/ql4_version.h b/drivers/scsi/qla4xxx/ql4_version.h
index faf173b..45e6cba 100644
--- a/drivers/scsi/qla4xxx/ql4_version.h
+++ b/drivers/scsi/qla4xxx/ql4_version.h
@@ -5,4 +5,4 @@
  * See LICENSE.qla4xxx for copyright and licensing details.
  */
 
-#define QLA4XXX_DRIVER_VERSION	"5.02.01.00.05.06-d1"
+#define QLA4XXX_DRIVER_VERSION	"5.02.03.00.05.06-d1"
